nvflare/private/fed/app/client/sub_worker_process.py

```diff
@@ -310,7 +310,7 @@ def stop(self):
 
 def main(args):
     workspace = Workspace(args.workspace, args.client_name)
-    app_custom_folder = workspace.get_client_custom_dir()
+    app_custom_folder = workspace.get_client_custom_dir(args.job_id)
```

Comment:
Incorrect! We need to add the custom dir of both "local" to sys path.
You should add another line to add the job's custom dir to the sys path.

Reply:
Changed the app_custom_folder to use get_app_custom_dir(args.job_id).